tests/test_slow_retrieval_attack.py

```diff
@@ -68,68 +68,20 @@
 repo_tool.disable_console_log_messages()
 
 
-class TestSlowRetrievalAttack(unittest_toolbox.Modified_TestCase):
 
-  @classmethod
-  def setUpClass(cls):
+def setUpTuf(obj, repo_name, mode):
```

Comment:
Two nits:
1. can we use snake_case here?
2. could we have a clearer name for this function? It seems to be setting up a copy of the test repo metadata, starting a server and creating an updater – `set_up_slow_retrieval_tests` is rather wordy but might suffice?

Reply:
`set_up_slow_retrieval_tests` will work as a title.